v1.1.0-dev1/Process.yml

```diff
@@ -555,27 +572,25 @@ $graph:
 
 - name: InputBinding
   type: record
-  abstract: true
   fields:
     - name: loadContents
       type:
         - "null"
         - boolean
       jsonldPredicate: "cwl:loadContents"
       doc: |
+        Use of `loadContents` in `InputBinding` is deprecated.
+        Preserved for v1.0 backwards compatability.  May be removed in
```

Comment:
"Will be removed in v2.0" and we should start that fork already (or at least a meta-issue)